imagestreams/postgresql-rhel.json

```diff
@@ -10,217 +10,203 @@
-          "description": "Provides a PostgreSQL 10 database on RHEL 8. For more information about using this database image, including OpenShift considerations, see https://github.com/sclorg/postgresql-container/blob/master/README.md.",
+          "description": "Provides a PostgreSQL 10 database on Ephemeral. For more information about using this database image, including OpenShift considerations, see https://github.com/sclorg/postgresql-container/blob/master/README.md.\n",
           "iconClass": "icon-postgresql",
           "tags": "database,postgresql",
           "version": "10"
         },
         "from": {
           "kind": "DockerImage",
-          "name": "registry.redhat.io/rhel8/postgresql-10:latest"
+          "name": "registry.redhat.io/rhscl/postgresql-10-rhel7:latest"
         },
         "referencePolicy": {
           "type": "Local"
         }
       },
       {
-        "name": "10-el7",
+        "name": "12-el7",
         "annotations": {
-          "openshift.io/display-name": "PostgreSQL 10 (RHEL 7)",
+          "openshift.io/display-name": "PostgreSQL 12 (Ephemeral)",
           "openshift.io/provider-display-name": "Red Hat, Inc.",
-          "description": "Provides a PostgreSQL 10 database on RHEL 7. For more information about using this database image, including OpenShift considerations, see https://github.com/sclorg/postgresql-container/blob/master/README.md.",
+          "description": "Provides a PostgreSQL 12 database on Ephemeral. For more information about using this database image, including OpenShift considerations, see https://github.com/sclorg/postgresql-container/blob/master/README.md.\n",
           "iconClass": "icon-postgresql",
           "tags": "database,postgresql",
-          "version": "10"
+          "version": "12"
         },
         "from": {
           "kind": "DockerImage",
-          "name": "registry.redhat.io/rhscl/postgresql-10-rhel7:latest"
+          "name": "registry.redhat.io/rhscl/postgresql-12-rhel7:latest"
         },
         "referencePolicy": {
           "type": "Local"
         }
       },
       {
-        "name": "10",
+        "name": "latest",
         "annotations": {
-          "openshift.io/display-name": "PostgreSQL (Ephemeral) 10",
+          "openshift.io/display-name": "PostgreSQL 15 (Latest)",
           "openshift.io/provider-display-name": "Red Hat, Inc.",
-          "description": "Provides a PostgreSQL 10 database on RHEL 7. For more information about using this database image, including OpenShift considerations, see https://github.com/sclorg/postgresql-container/blob/master/README.md.",
+          "description": "Provides a PostgreSQL 15 database on RHEL 8. For more information about using this database image, including OpenShift considerations, see https://github.com/sclorg/postgresql-container/blob/master/README.md.\n",
           "iconClass": "icon-postgresql",
-          "tags": "database,postgresql,hidden",
```

Comment:
Also this one tag is different in the Ephemeral streams.

Reply:
Discussed this with team, concluded that ephemeral streams are no longer needed and will be removed.